README.md

```diff
@@ -1,14 +1,53 @@
 # boozook
 Tools for extracting and editing Coktel Vision game resources
 
-# Compatible Games
-* Adi 4
+# Features
+* Decompile game scripts partially (TOT/0OT)
+* Extract and Rebuild game resources (STK)
+* Image extraction partially
+
+# Upcoming Features
+* extract images with correct colors.
+* decompile scripts such that all file have purpose and the output can be understood (TOT/0OT).
+* recompile Game scripts.
+
+# Suported games by boozook
+* Adi 4 (DEV6)
 * Adibou 1
-* Adibou 2
+* Adibou 2 (DEV6)
 * Geisha
 * Goblins 1
 * Gobliins 2
 * Gobliiins 3
-* Woodruff
+* Woodruff (DEV6)
 * Croustibat
 * Fascination
+
+## Installation
+[IMPORTANT] check if Python 3 is already installed on your System.
+1. Download the latest version of boozook.
+2. Download latest version of pakal.
```

Comment:
from here i would rather:
install poetry (https://python-poetry.org/)
install dependencies by running `poetry install`
activate virtual environment by running `poetry shell`

Reply:
does this can be get merged and i fix it tomorrow ?